doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc

```diff
@@ -5,191 +5,202 @@ This section highlights new features in {PRODUCT} {PRODUCT_VERSION}.
 
-=== Test scenarios
+ifdef::PAM[]
 
-The test scenarios designer in {CENTRAL} is now available with the following new features:
+=== Time series chart component on a page
 
-* You can now perform the undo and redo operations from the *Settings* panel. Also, a warning pop-up is displayed when you save or discard any changes from the *Settings* panel.
+It is now possible to use the *Time Series Chart* component to represent any time series data. You can create your own dashboards that can be connected to your time series data sets. You can also create, edit, and build a dashboard using a time series chart component. You can export the time series components to Dashbuilder Runtime and retrieve the information from a {KIE_SERVER} or any Prometheus data set.
 
-* In rule-based test scenarios, you can use the following new java classes as data objects:
+For more information, see {URL_MANAGING_SETTINGS}#assembly-creating-custom-pages[_{BUILDING_WIDGETS}_].
 
-** `LocalDateTime`
-** `LocalTime`
-** `BigDecimal`
-** `BigInteger`
+=== Dashbuilder Java API for dashboards
 
-* You can use `Byte`, `Character`, `Float`, `Long`, `Short`, `LocalDateTime`, `LocalTime`, `BigDecimal`, and `BigInteger` classes to create and manage the list of data object values.
+It is now possible to define dashboards using Dashbuilder Java API. You can create rows, columns, internal components, and external components and connect them to datasets.
 
-For more information, see {URL_DEVELOPING_DECISION_SERVICES}#assembly-test-scenarios[_{TESTING_DECISION_SERVICE}_].
+=== New `RecordsPerTransaction` parameter to use with `LogCleanupCommand`
 
-=== New parameter list for Decision Services
+You can now use the `RecordsPerTransaction` parameter with the `LogCleanupCommand` executor command that indicates the number of records in a transaction that is removed. For more information about `RecordsPerTransaction` parameter, see {URL_DEVELOPING_PROCESS_SERVICES}#assembly-designing-business-processes[_{DESIGNING_BUSINESS_PROCESSES}_].
 
-{PRODUCT} {ENTERPRISE_VERSION} includes a new *Decision Service details* section in the DMN editor properties panel containing a read-only list of *Input Data*, *Encapsulated Decisions*, and *Output Decisions*.
+=== Standalone BPMN editor now supports readonly mode
 
-For more information, see {URL_GETTING_STARTED}#assembly-getting-started-decision-services[_{GETTING_STARTED_DECISION_SERVICE}_] and {URL_DEVELOPING_DECISION_SERVICES}#assembly-dmn-models[_{DMN_MODELS}_].
+You can now use the standalone BPMN editor in readonly mode. For more information about standalone editors, see {URL_DEVELOPING_PROCESS_SERVICES}#assembly-designing-business-processes[_{DESIGNING_BUSINESS_PROCESSES}_].
 
-ifdef::PAM[]
+=== Support for work item definitions in VSCode extension and standalone editors
```

Comment:
This is about VSCode and standalone, don't think it makes sense under the "business central" category.

Reply:
Done